packages/pm/index.ts

```diff
@@ -0,0 +1,20 @@
+export * from './changeset'
+export * from './collab'
+export * from './commands'
+export * from './dropcursor'
+export * from './gapcursor'
+export * from './history'
+export * from './inputrules'
+export * from './keymap'
+export * from './markdown'
+export * from './menu'
+export * from './model'
+/** Since Module './markdown' has already exported a member named 'schema'.
```

Comment:
This was one of the main concerns why I kept the exports separated in their own paths - do we have any other overlapping imports that could conflict here?

Otherwise I really like to have everything available directly from `@tiptap/pm` as long as we also allow users to import via the old `@tiptap/pm/*` way to make it obvious which package users want to import.